packages/esm-patient-chart-app/src/visit/visits-widget/visit-detail-overview.scss

```diff
@@ -12,6 +12,17 @@
 .date {
   @include type.type-style('body-compact-01');
   color: $text-02;
+  padding-right: 10px;
```

Comment:
Could we convert these into `rem` units? Other wise, this looks good to me.

Reply:
Done Ian, sorry for the wrong name in the second commit.